core/server/worker/src/main/java/alluxio/worker/block/BlockLockManager.java

```diff
@@ -190,29 +191,41 @@ public void unlockBlock(long lockId) throws BlockDoesNotExistException {
       }
     }
     unlock(lock, record.getBlockId());
+    return true;
+  }
+
+  /**
+   * Releases the lock with the specified lock id.
+   *
+   * @param lockId the id of the lock to release
+   * @throws BlockDoesNotExistException if lock id cannot be found
+   */
+  public void unlockBlock(long lockId) throws BlockDoesNotExistException {
```

Comment:
possible to consolidate this one with `public boolean unlockBlockNoException(long lockId)` too?